webcompat/static/css/development/components/UI.css

```diff
@@ -0,0 +1,79 @@
+/** @define UI */
+
+html {
+  -ms-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
```

Comment:
No `-moz-text-size-adjust`? :p (I guess the webkit alias will just work anyways, in the future...)